File: test_fal.py
import os
import fal_client

def test_fal_model(model_id):
    print(f"Testing {model_id}...")
    try:
        url = "https://hbbody.com/wp-content/uploads/2023/06/WEB-505SPR.png.png"
        result = fal_client.subscribe(
            model_id,
            arguments={"image_url": url},
            with_logs=False
        )
        print("Success:", result['image']['url'] if 'image' in result else result)
    except Exception as e:
        print(f"Failed {model_id}: {e}")

if __name__ == "__main__":
    test_fal_model("fal-ai/imageutils/rembg")
    test_fal_model("briaai/rmbg-1.4")
    test_fal_model("bria/background/remove")

File: test_fal.js
async function test() {
    const url = "https://fal.run/fal-ai/imageutils/rembg";
    const res = await fetch(url, {
        method: "POST",
        headers: {
            "Authorization": "Key c21a3057-f51b-4768-b37c-680c8eaf93f2:7539b7de3124f8080d6b2316ab2921d7",
            "Content-Type": "application/json"
        },
        body: JSON.stringify({ image_url: "https://hbbody.com/wp-content/uploads/2023/06/WEB-505SPR.png.png" })
    });
    console.log("fal-ai/imageutils/rembg: ", await res.json());

    const url2 = "https://fal.run/briaai/rmbg-1.4";
    const res2 = await fetch(url2, {
        method: "POST",
        headers: {
            "Authorization": "Key c21a3057-f51b-4768-b37c-680c8eaf93f2:7539b7de3124f8080d6b2316ab2921d7",
            "Content-Type": "application/json"
        },
        body: JSON.stringify({ image_url: "https://hbbody.com/wp-content/uploads/2023/06/WEB-505SPR.png.png" })
    });
    console.log("briaai/rmbg-1.4: ", await res2.json());
}
test();
